cost-analyzer/charts/prometheus/templates/server-ingress.yaml

```diff
@@ -5,7 +5,9 @@
 {{- $serviceName := include "prometheus.server.fullname" . }}
 {{- $servicePort := .Values.server.service.servicePort -}}
 {{- $extraPaths := .Values.server.ingress.extraPaths -}}
-{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
+{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1" }}
+apiVersion: networking.k8s.io/v1
+{{ else if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
```

Comment:
Why do we compatibility check on this but not the rbac resources?

Reply:
I believe the RBAC goes back beyond the version we support ( k8s 1.11)